src/components/picker/index.tsx

```diff
@@ -120,6 +136,7 @@ const Picker = React.forwardRef((props: PickerProps, ref) => {
     pickerExpandableRef: pickerExpandable,
     getItemValue,
     topBarProps,
+    headerProps: type.modal && headerProps,
```

Comment:
Why you need to pass the headerProps?

Reply:
using the `headerProps` `onCancel` prop.

Reply:
See my comment below - why not simply have a onCancel prop?